src/otx/recipe/detection/rtmdet_tiny.yaml

```diff
@@ -60,10 +60,21 @@ overrides:
         - class_path: otx.core.data.transform_libs.torchvision.RandomCrop
           init_args:
             crop_size: $(input_size)
+        - class_path: otx.core.data.transform_libs.torchvision.RandomAffine
+          enable: false
+        - class_path: otx.core.data.transform_libs.torchvision.PhotoMetricDistortion
+          enable: false
         - class_path: otx.core.data.transform_libs.torchvision.YOLOXHSVRandomAug
         - class_path: otx.core.data.transform_libs.torchvision.RandomFlip
           init_args:
             prob: 0.5
+            is_numpy_to_tvtensor: true
+        - class_path: torchvision.transforms.v2.RandomVerticalFlip
```

Comment:
I still think that it is better to use horizontal flip, since vertical is not natural and not common in the CV field

Reply:
It's a misunderstanding. We keep h-flip as default as-is, and adding v-flip as optional aug,
and it's been included both in Roboflow & Landing.AI apps as well.